modules/client/src/test/java/org/apache/ignite/internal/client/RepeatedFinishClientTransactionTest.java

```diff
@@ -220,7 +220,7 @@ public void testEnlistFailAfterCommit() {
 
         tx.commit();
 
-        WriteContext wc = new WriteContext(emptyTracker());
+        WriteContext wc = new WriteContext(emptyTracker(), 1);
```

Comment:
It might be better to use a constant from `ClientOp`.

Reply:
🆗